### PR TITLE
HW-5.2

### PR DESCRIPTION
**Pull request by vremere-a**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -8,4 +8,79 @@
...
+    public double getPower() {
+        return power;
+    }
+
+    public double getSpeed() {
+        return speed;
+    }
+
+    public static class PlaneBuilder {
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: redundant empty line

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -8,4 +8,79 @@
 
 public class Plane {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: redundant empty line

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java (hunk 1 above)
Comment (vremere-a) [reply src/main/java/core/basesyntax/Plane.java]: done

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (vremere-a) [reply src/main/java/core/basesyntax/Plane.java]: done

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,78 @@
  */
 
 public class Plane {
+    private String name;
+    private String type;
+    private String color;
+    private double power;
+    private double speed;
 
+    public Plane(PlaneBuilder planeBuilder) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: check https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html about this one

Comment (vremere-a) [reply src/main/java/core/basesyntax/Plane.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**